treeherder/etl/perf.py

```diff
@@ -271,6 +272,13 @@ def _load_perf_datum(job: Job, perf_datum: dict):
                 push_timestamp=deduced_timestamp,
                 defaults={'value': value[0], 'application_version': application_version},
             )
+
+            if subtest.get("replicates", []) is not None and len(subtest.get("replicates", [])) > 0:
```

Comment:
`subtest.get("replicates", [])` wil never be None, at most `[]`
```suggestion
            if subtest.get("replicates", []) and len(subtest.get("replicates", [])) > 0:
```

Reply:
Fixed, thanks!